examples/stage/01_odc_eks/provider.tf

```diff
@@ -1,4 +1,18 @@
 provider "aws" {
-  region      = local.region
+  region                 = local.region
```

Comment:
@NikitaGandhi I am wondering whether it might be a good idea to introduce a variables.tf section in each stage so that the region, state bucket and state files can be supplied there instead of in different files like it is now. Then we won't need the local variables like above

Reply:
What would be the best alternative? I would have thought perhaps having a variables section could remove this issue, and it could also be used to store the state file information in as well, however that isn't possible either